format/spec.md

```diff
@@ -632,6 +632,29 @@ When expiring snapshots, retention policies in table and snapshot references are
     2. The snapshot is not one of the first `min-snapshots-to-keep` in the branch (including the branch's referenced snapshot)
 5. Expire any snapshot not in the set of snapshots to retain.
 
+#### Snapshot statistics
```

Comment:
Can you move this to be `Table statistics` just below `Table Metadata Fields`? I think that's a better place since these aren't part of a snapshot.